akka-distributed-data/src/main/scala/akka/cluster/ddata/ORSet.scala

```diff
@@ -337,11 +339,18 @@ final class ORSet[A] private[akka] (
   /**
    * Removes an element from the set.
    */
-  def -(element: A)(implicit node: Cluster): ORSet[A] = remove(node, element)
+  def remove(element: A)(implicit node: SelfUniqueAddress): ORSet[A] = remove(node.uniqueAddress, element)
```

Comment:
also
```
def remove(node: SelfUniqueAddress, element: A): ORSet[A]
```

for Java API

Reply:
Done:  [ORSet.scala](https://github.com/akka/akka/blob/eb5772dfbe58d1ea1eb101a424c49a8c12bd6131/akka-distributed-data/src/main/scala/akka/cluster/ddata/ORSet.scala#L349)